submissions/Shrai-dev/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,91 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: 'dog',
+   name: 'Buddy',
+   gender: 'male',
+   age: 3,
+   legs: 4,
+   hands: 0,
+   saying: 'bark-bark!',
+   friends: ['Elsa', 'Chip', 'Max']
+};
+
+const cat = {
+   species: 'cat',
+   name: 'Chloe',
+   gender: 'female',
+   age: 1,
+   legs: 4,
+   hands: 0,
+   saying: 'meow!',
+   friends: ['Leo', 'Simon', 'Kitty']
+};
+
+const man = {
+   species: 'human',
+   name: 'Brian',
+   gender: 'male',
+   age: 26,
+   legs: 2,
+   hands: 2,
+   saying: 'Everyone thinks of changing the world, but no one thinks of changing himself!',
+   friends: ['Ethan', 'James', 'Ryan']
+};
+
+const woman = {
+   species: 'human',
+   name: 'Olivia',
+   gender: 'female',
+   age: 24,
+   legs: 2,
+   hands: 2,
+   saying: 'The simplest way to be happy is to do good!',
+   friends: ['Maria', 'Kelly', 'Jennifer']
+}
+
+// ======== OUTPUT ========
+
+const inhabitants = [dog, cat, man, woman];
+
+const message = ({
+   species,
+   name,
+   age,
+   legs,
+   hands,
+   friends,
+   saying
+}) => {
+   return 'I am a ' + `${species}` + '. My name is ' + `${name}` + '. I am ' + `${age}` + ' years old. I have ' + `${legs}` + ' legs and ' + `${hands}` + ' hands. My friends are: ' + `${friends}` + '. I say: ' + `${saying}`;
+}
```

Comment:
The options below are equivalent and both are more cosistent than the current approach combining string concatenation and template strings/literals:
- ` 'I am a ' + species + '. My name is ' + name `
- `` `I am ${species}. My name is ${name}` ``

The latter is generally preferred. Check [the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to master it.

Reply:
Done.